### DISCUSSION TITLE
Multiple metrics for model evaluation

### DISCUSSION THREAD

**Discussion by gintriago12** [online-ml/river#700] [Q&A] [answered]
Hi,

Is there a way to use multiple metrics for model evaluation? I mean something like this:

metric1 = metrics.Accuracy()
metric2 = metrics.roc_auc
evaluate.progressive_val_score(dataset, model, metric=metric1+metric2)

or

metric1 = metrics.Accuracy()
metric2 = metrics.roc_auc
metric = compose.TransformerUnion(metric1)
evaluate.progressive_val_score(dataset, model, metric)

Thanks.

Best,
Gabriel

**Comment by MaxHalford** [comment, marked as answer]
Yep, here you go:

```py
metrics = metrics.Accuracy() + metrics.ROCAUC()
evaluate.progressive_val_score(dataset, model, metrics)
```
